ecommerce/frontend/src/utils.js

```diff
@@ -105,3 +105,14 @@ export function formatProduct({
         brand,
     }
 }
+
+/**
+ * Round half away from zero ('commercial' rounding)
+ * Uses correction to offset floating-point inaccuracies.
+ * Works symmetrically for positive and negative numbers.
+ */
+export function round(num, decimalPlaces = 2) {
```

Comment:
Really useful for prices all over the frontend. I think you should tell everyone on the team that this function exists in `utils.js`

Reply:
Sure I will announce it.